tests/shutdown/reboot.pm

```diff
@@ -0,0 +1,24 @@
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Reboot, then verify boot completed.
+#
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use power_action_utils 'power_action';
+
+sub run {
+    my $self = shift;
+    power_action('reboot', textmode => 1);
```

Comment:
we should keep the comment as well:
`# DESKTOP can be gnome, but patch is happening in shell, thus always force reboot in shell`